.github/workflows/build.yml

```diff
@@ -31,13 +31,10 @@ jobs:
           asset_name: "cppsdk-${{ env.CPP_SDK_TAG }}-${{ env.BUILD_PLATFORM }}.zip"
```

Comment:
Can't comment on the env directly but it should become this right?
```
env:
      BUILD_PLATFORM: 'macos64-universal'
```

Reply:
or just append `-universal` after teh ${{ env.BUILD_PLATFORM }}

Reply:
I'll take a look at this when SDK is available.

Reply:
fair point.